Mathlib/Tactic/Lint/Basic.lean

```diff
@@ -34,14 +34,14 @@ initialize nolintAttr : ParametricAttribute (Array Name) ←
     getParam := fun decl stx =>
       match stx with
         -- TODO: validate linter names
-        | `(attr|nolint $[$ids]*) => ids.mapM (·.getId.eraseMacroScopes)
+        | `(attr|nolint $[$ids]*) => pure $ ids.map (·.getId.eraseMacroScopes)
         | _ => throwError "unexpected nolint syntax {stx}"
   }
 
 /-- Returns true if `decl` should be checked
 using `linter`, i.e., if there is no `nolint` attribute. -/
 def shouldBeLinted [Monad m] [MonadEnv m] (linter : Name) (decl : Name) : m Bool := do
-  !((← nolintAttr.getParam (← getEnv) decl).getD {}).contains linter
+  pure !((nolintAttr.getParam (← getEnv) decl).getD {}).contains linter
```

Comment:
@gebner Also check out the funky precedence here. `!` has precedence 40 on the right and `max` on the left, so it acts like `$` here.

Reply:
omg